pkgs/development/python-modules/mnist/default.nix

```
@@ -0,0 +1,25 @@
+{ buildPythonPackage, fetchPypi, lib, numpy }:
+
+buildPythonPackage rec {
+  pname = "mnist";
+  version = "0.2.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0m3vz9z7gay8nndid2784j95fzif502shfvik3f8mggvx6nlz4r4";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+  ];
+
+  meta = with lib; {
+   description = "MNIST dataset";
```

```suggestion
   description = "Python utilities to download and parse the MNIST dataset";
```
no longDescription needed